packages/dev/core/src/Cameras/targetCamera.ts

```diff
@@ -255,6 +255,7 @@ export class TargetCamera extends Camera {
      * @param target Defines the new target as a Vector
      */
     public setTarget(target: Vector3): void {
+        // REVIEW: why is this here?
         this.upVector.normalize();

```

Comment:
Can anyone explain why we need this here in this function? It's not used in this function, so why would we normalize here? It seems like we should normalize when we actually use it?